design-doc-template.adoc

```diff
@@ -82,73 +94,98 @@ __<List the installation types that are relevant for the features and remove any
+////
 
 === Default Configuration
 
-__<Does the proposed work change the default value of any current configuration attributes? Does it change the
-configuration generated by any current Galleon layers?>__
+////
+Does the proposed work change the default value of any current configuration attributes? Does it change the
+configuration generated by any current Galleon layers?
+////
 
 === Importing Existing Configuration
 
-__<Does the proposed work affect the ability to run WildFly running an existing configuration? Is there anything else
-about the proposed work that may require changes to the WildFly server migration tool?>__
+////
+Does the proposed work affect the ability to run WildFly running an existing configuration? Is there anything else
+about the proposed work that may require changes to the WildFly server migration tool?
+////
 
 === Deployments
 
-__<Does this feature change the behavior of deployments in incompatible ways? If yes, please detail what is the deployment
-issue observed when no change is done, and what is the change needed to solve the deployment issue.>__
+////
+Does this feature change the behavior of deployments in incompatible ways? If yes, please detail what is the deployment
+issue observed when no change is done, and what is the change needed to solve the deployment issue.
+////
 
 === Interoperability
 
-__<Is this feature impacting interoperability?>__
+////
+Is this feature impacting interoperability?
+////
 
 == Implementation Plan
 
-__<This section is optional. If you have a complex feature which cannot be delivered all in one go, suggest the strategy.>__
+////
+This section is optional. If you have a complex feature which cannot be delivered all in one go, suggest the strategy.
+////
 
 == Admin Clients
 
-__<Identify the level of compatibility this feature will have with the existing admin clients (JBoss CLI and the Admin
-Console / HAL). Identify any follow-up work that will be required in the clients and link issues created to track this work.>__
+////
+Identify the level of compatibility this feature will have with the existing admin clients (JBoss CLI and the Admin
+Console / HAL). Identify any follow-up work that will be required in the clients and link issues created to track this work.
+////
 
 == Security Considerations
 
-__<What impact on security does this feature have?>__
+////
+What impact on security does this feature have?
+////
 
 [[test_plan]]
 == Test Plan
 
-__<Depending on the selected stability level, the appropriate section below should be completed, including a brief
-description of how testing is to be performed in accordance with the selected stability level. The non-relevant sections
-may be removed as needed.>__
 ////
+Depending on the selected stability level, the appropriate section below should be completed, including a brief
+description of how testing is to be performed in accordance with the selected stability level. The non-relevant sections
+may be removed as needed.
+
```

Comment:
This feels more like something that should be deleted, but since it's a preface to a whole block of adoc commentary I think having it be adoc as well is fine.